README.md

```diff
@@ -3,16 +3,35 @@
 ![freedy](https://i.imgur.com/Z0P5UVL.jpg)
 
 ## What is Freedy?
-Freedy is a free and simple RSS/ Atom reader for the browser, written in Python with Flask. This is our first web application, so if there are any fatal mistakes, feel free to tell us!
+Freedy is a free and simple RSS/ Atom reader for the browser, written in Python with Flask. Using Bootstrap as a mobile-first web framework, we were able to create a nice looking webapp for every kind of device.
 
 ## Features
-- Small footprint on performance
+- Looks great on tablets, phones etc. thanks to Bootstrap.
+- Compatible with RSS and Atom feeds.
 - Add as many feeds as you like!
-- Amount of the recent posts shown of each feed can be configured
+- Amount of the recent posts shown of each feed can be configured.
+- ... and of course the simple deployment via Docker ;)
 
 ## Usage
-You basically just need to clone the repo, create a virtualenv for the project and then run the flask/gunicorn server. We also recommend using a reverse proxy (eg. nginx) in front of the app.
+You basically just need to clone the repo, setup the virtualenv for the project and then run the flask/gunicorn server. We also recommend using a reverse proxy (eg. nginx) in front of the app for security reasons.
```

Comment:
TODO: Add instructions on how to add URLs

Reply:
I'll get right to it.